### PR TITLE
feat(types): implement New... constructors for button types

### PR DESCRIPTION
**Pull request by chicksick**

### CODE REVIEW

File: types.go [hunk 1]
```diff
@@ -1009,6 +1009,16 @@ type InlineKeyboardMarkup struct {
 	InlineKeyboard [][]InlineKeyboardButton `json:"inline_keyboard"`
 }
 
+func NewInlineKeyboardMarkup(rows ...[]InlineKeyboardButton) InlineKeyboardMarkup {
+	var keyboard [][]InlineKeyboardButton
+
+	keyboard = append(keyboard, rows...)
```
Comment (mr-linch) [inline types.go]: - [x] make type InlineKeyboardRow
- [x] use preallocated array

**Review by mr-linch — CHANGES_REQUESTED** [review]

File: client_bot_test.go [hunk 2]
```diff
@@ -44,9 +44,6 @@ func TestClient_GetMe(t *testing.T) {
 		assert.True(t, user.IsBot)
 		assert.NotZero(t, user.FirstName)
 		assert.NotZero(t, user.Username)
-		assert.True(t, user.SupportsInlineQueries)
-		assert.True(t, user.CanReadAllGroupMessages)
```
Comment (mr-linch) [inline client_bot_test.go]: set this parameters with your bot

File: types.go [hunk 3]
```diff
@@ -1006,7 +1006,27 @@ type ReplyKeyboardRemove struct {
...
+	return row
+}
+
+func NewInlineKeyboardMarkup(rows ...InlineKeyboardRow) InlineKeyboardMarkup {
+	keyboard := make([]InlineKeyboardRow, 0, len(rows))
+
+	keyboard = append(keyboard, rows...)
+
+	return InlineKeyboardMarkup{
+		InlineKeyboard: keyboard,
```
Comment (mr-linch) [inline types.go]: why not just pass rows here?

**Review by mr-linch — CHANGES_REQUESTED** [review]

File: types.go (hunk 3 above)
Comment (chicksick) [reply types.go]: забыл поменять

File: types.go [hunk 4]
```diff
@@ -1006,7 +1006,23 @@ type ReplyKeyboardRemove struct {
 // InlineKeyboardMarkup object represents an inline keyboard that appears right next to the message it belongs to.
 type InlineKeyboardMarkup struct {
 	// Array of button rows, each represented by an Array of InlineKeyboardButton objects
-	InlineKeyboard [][]InlineKeyboardButton `json:"inline_keyboard"`
+	InlineKeyboard []InlineKeyboardRow `json:"inline_keyboard"`
+}
+
+type InlineKeyboardRow []InlineKeyboardButton
+
+func NewInlineKeyboardRow(buttons ...InlineKeyboardButton) InlineKeyboardRow {
```
Comment (mr-linch) [inline types.go:1014]: maybe just return buttons?

**Review by mr-linch — CHANGES_REQUESTED** [review]

**Review by mr-linch — APPROVED** [review]

**Pull request merged**